apps/dashboard/src/modules/attendance/components/attendance-page/utils.ts

```diff
@@ -0,0 +1,5 @@
+export const rangeToString = (ranges: number[][]): string => {
+  // example: [1,2] [2,3]  => 1,2,3
+  const flat = ranges.flat()
+  return flat.sort().join(", ")
+}
```

Comment:
While it shouldn't matter too much for our numbers, this is a subtle javascript bug.

Array.sort sorts alphanumerically, which means that [10, 2].sort() yields [10, 2].

I also think it could use some better naming, but not sure what's appropriate